Library/Homebrew/build.rb

```diff
@@ -245,6 +246,9 @@ def normalize_pod2man_outputs!(formula)
   formula = args.named.to_formulae.first
   options = Options.create(args.flags_only)
   build   = Build.new(formula, options, args:)
+
+  Pathname.prepend WriteMkpathExtension
```

Comment:
This needs to be done for `brew test` too:

```
  Error: mikutter: failed
  An exception occurred within a child process:
    Errno::ENOENT: No such file or directory @ rb_sysopen - /private/tmp/mikutter-test-20250825-45320-uw8qyn/.mikutter/plugin/test_plugin/test_plugin.rb
```

https://github.com/Homebrew/homebrew-core/actions/runs/17222931612/job/48871557746?pr=234917#step:3:880